Test_App/src/main/java/Test_App/UI/AddPageController.java

```diff
@@ -113,11 +120,25 @@ public void goBack(ActionEvent event) {
     public void add(ActionEvent event) throws IOException {
 
         if (!expenseRadioButton.isSelected() && !incomeRadioButton.isSelected()) {
-            Alert errorAlert = new Alert(Alert.AlertType.ERROR, "Please select transaction type.", ButtonType.OK);
-            errorAlert.showAndWait();
+            new Utils().showAlert(Alert.AlertType.ERROR, "Please select transaction type.");
```

Comment:
showAlert() method could be static which will help us to avoid having a lot of created objects Utils (produced by new Utils())